peachjam/forms.py

```diff
@@ -140,6 +141,9 @@ def filter_queryset(self, queryset, exclude=None):
         if outcomes and exclude != "outcomes":
             queryset = queryset.filter(outcomes__name__in=outcomes).distinct()
 
+        if q and exclude != "q":
```

Comment:
This doesn't work as we need it to. Currently, this will mean that the filter is applied when the facet values are calculated, which we don't want. The filtering must come entirely after the faceting is done. It's a post-aggregation filter.